src/main/java/org/generationcp/middleware/dao/dms/DmsProjectDao.java

```diff
@@ -809,7 +809,7 @@ public StudyMetadata getStudyMetadataForGeolocationId(final Integer geolocationI
 				return null;
 			}
 		} catch (final HibernateException e) {
-			final String message = "Error with getStudyMetadataForGeolocationId() query from study with geoloCationId: " + geolocationId;
+			final String message = "Error with getStudyMetadataForGeolocationId() query from study with geoloCationId: " + instanceId;
```

Comment:
Rename method in error message to: `getStudyMetadataForInstanceId()`

Reply:
Done